OmniBLE/PumpManagerUI/ViewModels/OmniBLESettingsViewModel.swift

```diff
@@ -57,6 +57,16 @@ class OmniBLESettingsViewModel: ObservableObject {
         }
     }
 
+    var serviceTimeRemainingString: String? {
+        if let serviceTimeRemaining = pumpManager.podServiceTimeRemaining {
+            timeRemainingFormatter.allowedUnits = serviceTimeRemaining < .hours(2) ? [.hour, .minute] : [.hour]
+            if let serviceTimeRemainingString = timeRemainingFormatter.string(from: serviceTimeRemaining)?.replacingOccurrences(of: ",", with: "") {
```

Comment:
I think keeping the separator in there is fine, both from an english standpoint, and to avoid messing with other localizations inadvertently.

Reply:
Definitely agree with this one.  The only localizations supported by Loop that put a date component separator between hours and minutes are English, Swedish, and Vietnamese, and they all use a comma, but I agree that perfect consistency between this and the lifecycleProgress view is not necessary and will remove it.